### PR TITLE
Fixed #2385 - Fixed Calendar Dashlet Activities

### LINKED ISSUE #2385: Calendar Dashlet not usable in SuiteCRM 7.7.5
Calendar Dashlet in Google Chrome is almost unusable. Meetings cannot be scheduled because click on time slot before late PMs does nothing. Clicking on late PMs does produce new meeting dialogue but the time is wrong (pls see attached screenshot).
#### Expected Behavior

I should be able to click on time slot and schedule a meeting with a correct date & time.
#### Steps to Reproduce
1. Log into online demo
2. Go to Home module
3. Add Calendar dashlet if not displayed
4. Try to schedule meeting
#### Your Environment

- SuiteCRM Version used:
- Google Chrome Version 53.0.2785.101 m (64-bit)
- SuiteCRM official demo environment
- Windows 7 Enterprise 64bit Service Pack 1 OS

Screenshot: [https://postimg.org/image/bl21bmu5n/](https://postimg.org/image/bl21bmu5n/)

### PR DESCRIPTION
**Pull request by daniel-samson**
In relation to #2385.

## Description

On the calendar dashlet, the position of the activities are not correctly rendered . This is due to the time taken to load JavaScript. I have added a time out to call the fullcalendar.io plugin 300ms later, in order to give the browser time to process the JavaScript.

## Motivation and Context

Fixes the issue that prevents users from creating calendar events in the dashboard.

## How To Test This

- Try and add an activity to the calendar dashlet

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: jssource/src_files/modules/Calendar/Cal.js [hunk 1]
```diff
@@ -673,7 +673,7 @@ CAL.dialog_save = function () {
       ajaxStatus.hideStatus();
     }
   };
-  var url = "index.php?module=Calendar&action=SaveActivity&sugar_body_only=true";
+  var url = "index.php?module=Calendar&action=SaveActivity&sugar_body_only=true&XDEBUG_SESSION_START=13537";
```
Comment (samus-aran) [inline jssource/src_files/modules/Calendar/Cal.js]: Remove Debug

File: jssource/src_files/modules/Calendar/Cal.js [hunk 2]
```diff
@@ -737,6 +737,11 @@ CAL.refresh = function () {
   YAHOO.util.Connect.asyncRequest('POST', url, callback, CAL.toURI(data));
   CAL.clear();
 }
+
+CAL.fixEventPosition = function() {
```
Comment (samus-aran) [inline jssource/src_files/modules/Calendar/Cal.js]: Where is this being called?  Looks like a redundant function.

**Comment by daniel-samson** [comment]
Changes updated

**Comment by samus-aran** [comment]
Still think that having a timeout is hacky.  Must be something causing the height not to be rendered correctly - try again.

**Comment by daniel-samson** [comment]
Updated pr.

**Pull request merged**